### PR TITLE
Convert Attribution code block to tables

### PR DESCRIPTION
**Pull request by wopian**
Migrated the attribution section to tables for all languages to aid readability (particularly the approval list) as everything is ALL CAPS.

Unfortunately GFM doesn't allow header-less tables, so it's either an empty header:

or copyright as a header:

### CODE REVIEW

File: README.zh_cn.md [hunk 1]
```diff
@@ -18,80 +18,50 @@
...
+Margaret H. Hamilton  | Colossus Programming Leader<br>Apollo Guidance and Navigation |  1969年3月28日
+
+核准人              | Role | 日期
+:----------------- | :--- | :---
+Daniel J. Lickly   | Director, Mission Program Development<br>Apollo Guidance and Navigation Program | 1969年3月28日
+Fred H. Martin     | Colossus Project Manager<br>Apollo Guidance and Navigation Program |  1969年3月28日
+Norman E. Sears    | Director, Mission Development<br>Apollo Guidance and Navigation Program | 1969年3月28日
+Richard H. Battin  | Director, Mission Development<br>Apollo Guidance and Navigation Program | 1969年3月28日
+David G. Hoag      | Director<br>Apollo Guidance and Navigation Program | 1969年3月28日
+Ralph R. Ragan     | Deputy Director<br>Instrumentation Laboratory | 1969年3月28日
```
Comment (oldmud0) [inline README.zh_cn.md]: These fields were already localized in the target language, but they were pasted in English.

Comment (chrislgarry) [reply README.zh_cn.md]: Hmm looks so.

**Comment by chrislgarry** [comment]
Awesome work. Infinitely more readable. Lets have a look at @oldmud0 's comment and then we can merge.

**Comment by wopian** [comment]
Roles rows did not kept their translations as the text was in all-caps and I don't know the capitalisation rules. 

`Comanche055` and `Luminary099` are both derived from the `Purpose` in the original text and reworded, which without knowing the language I can't do without mutilating the translation. Which is better than a translation that no longer matches the source.

If this is 🆗'd the people who did these translations can get mentioned to do the modifications correctly

File: README.ko_kr.md [hunk 2]
```diff
@@ -28,74 +28,38 @@ PR을 열기 전에 [CONTRIBUTING.md][7] 을 읽어보시기 바랍니다.
...
-#
-#   이 프로그램은 R-577에 명세된 사령선 모듈에 사용됩니다.            
-#   DSR 프로젝트 55-23870 에 의해 준비되었으며, NASA 유인 우주선 센터가               
-#   MIT 기계 연구소 와의 NAS 9-4065 계약에 의해 스폰싱 하였습니다.
-#
-#************************************************************************
-
-
-제출됨: MARGARET H. HAMILTON           일시:    1969년 3월 28일
-    M.H.HAMILTON, 콜로서스 프로그래밍 리더
```
Comment (chrislgarry) [inline README.ko_kr.md]: We can copy and paste the roles from the original READMEs made by the translators, such as right here. I think "콜로서스 프로그래밍 리더" is the role. Or am I misunderstanding the issue.

Comment (wopian) [reply README.ko_kr.md]: The Korean readme already has the translated roles in this PR

**Comment by chrislgarry** [comment]
@wopian this still good?

**Comment by wopian** [comment]
Good with me 👍

**Pull request merged**